docs/external-classes.json

```diff
@@ -59,23 +59,17 @@
   "name": "Google\\Auth\\",
   "uri": "https://github.com/google/google-auth-library-php/blob/master/src/%s.php"
 }, {
-  "name": "Google\\GAX\\",
-  "uri": "https://googleapis.github.io/gax-php/#Google/GAX/%s.html"
+  "name": "Google\\ApiCore\\",
+  "uri": "https://googleapis.github.io/gax-php/#Google/ApiCore/%s.html"
 }, {
   "name": "Google\\Api\\",
   "uri": "https://googleapis.github.io/gax-php/#Google/Api/%s.html"
-}, {
-  "name": "Google\\Cloud\\Audit\\",
```

Comment:
@dwsupplee @jdpedrie I removed this because I believe it is unnecessary, since we have and entry that maps `Google\Cloud`. Is that correct? Or do we still need this?

Reply:
You're right, it can go. I just verified it will generate the correct docs link.